boba-chain-ops/ether/migrate_test.go

```diff
@@ -0,0 +1,425 @@
+package ether
+
+import (
+	"math/big"
+	"math/rand"
+	"reflect"
+	"testing"
+
+	"github.com/ledgerwatch/erigon-lib/chain"
+	"github.com/ledgerwatch/erigon-lib/common"
+	"github.com/ledgerwatch/erigon/boba-bindings/predeploys"
+	"github.com/ledgerwatch/erigon/boba-chain-ops/crossdomain"
+	"github.com/ledgerwatch/erigon/core/types"
+	"github.com/stretchr/testify/require"
+)
+
+func TestSetBalance(t *testing.T) {
+	g := &types.Genesis{
+		Alloc: types.GenesisAlloc{
+			predeploys.LegacyERC20ETHAddr: types.GenesisAccount{
+				Storage: map[common.Hash]common.Hash{
+					CalcOVMETHStorageKey(common.Address{1}):                       {1},
+					CalcOVMETHStorageKey(common.Address{2}):                       {2},
+					CalcAllowanceStorageKey(common.Address{1}, common.Address{2}): {3},
+				},
+			},
+		},
+	}
+	SetBalance(g, common.Address{1}, big.NewInt(1), CalcOVMETHStorageKey(common.Address{1}))
+	accountState := types.GenesisAccount{
+		Balance: big.NewInt(1),
+	}
+	storageState := map[common.Hash]common.Hash{
+		CalcOVMETHStorageKey(common.Address{2}):                       {2},
+		CalcAllowanceStorageKey(common.Address{1}, common.Address{2}): {3},
+	}
+	if !reflect.DeepEqual(accountState, g.Alloc[common.Address{1}]) {
```

Comment:
Given that we're already using testify through much of the rest of the code, I think it would make more sense for us to use a:

```
require.Equal(t, accountState, g.Alloc[common.Address{1}])
```

This will eliminate the need to try to do our own formatting on not equal, and, it will more reliably handle some cases where reflect.DeepEqual isn't actually what you want.  (This comment holds through the tests)

Reply:
Sounds good.